src/app/personside/infotabs/oversikt/Oversikt.tsx

```diff
@@ -11,16 +11,23 @@ import { INFOTABS } from '../InfoTabEnum';
 import SakerOversikt from './SakerOversikt';
 
 const Style = styled.article`
-    display: flex;
+    @media (${theme.media.wideScreen}) {
+        display: flex;
+        justify-content: center;
+    }
 `;
 
 const KolonneStyle = styled.div`
     > * {
         margin-bottom: ${theme.margin.px30};
     }
-    &:not(:last-child) {
-        margin-right: ${theme.margin.px30};
+    @media (${theme.media.wideScreen}) {
+        &:not(:last-child) {
+            margin-right: ${theme.margin.px30};
+        }
     }
+    flex-basis: 50%;
+    max-width: ${pxToRem(800)};
```

Comment:
Hvorfor sette max-width?

Reply:
Fordi oversikten blir ganske ubrukelig om den blir strekt ut for mye ;-)